src/uploadGraphqlSchemaCommand.ts

```diff
@@ -0,0 +1,52 @@
+import vscode from 'vscode';
+
+export default (
+  mode: string = 'merge',
+  adminSecretKey: string,
+  outputChannel: vscode.OutputChannel
+) => async () => {
+  const { activeTextEditor } = vscode.window;
+
+  if (!activeTextEditor || activeTextEditor.document.languageId !== 'graphql') {
+    vscode.window.showWarningMessage(
+      'You have to select a Graphql document to upload a schema.'
```

Comment:
'You must select a GraphQL..'

Reply:
Thanks for the suggestion! Changed this and also added hints about the file extensions both (`.graphql` and `.gql`)